jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/diagram/ProcessInstanceDiagramPresenter.java

```diff
@@ -150,11 +159,12 @@ protected void loadProcessInstanceDetails() {
 
             view.setTimerInstances(timerInstances);
 
+            view.setShowOrHideNodeActionsCommand(() -> showOrHideNodeActions(forLog, processInstance));
+
             timers = summary.getProcessDefinition().getTimers().stream().collect(toList());
 
-            if (forLog || processInstance.getState() != ProcessInstance.STATE_ACTIVE) {
-                view.hideNodeActions();
-            }
+            view.showOrHideNodeActionsTriggered();
+            expandDiagram(processInstance, summary);
```

Comment:
@bxf12315 Here I think we have to disable the expand if there are subprocess or parent pi to show.. 
I would remove the new "protected void expandDiagram(ProcessInstanceSummary processInstance, ProcessInstanceDiagramSummary summary) { .." and replace the call by
            if (forLog || processInstance.getState() != ProcessInstance.STATE_ACTIVE) {
                view.expandDiagram();
                view.disableExpandAnchor();
            }

            if(!summary.getSubProcessInstances().isEmpty() || summary.getParentProcessInstanceSummary() != null){
                view.disableExpandAnchor();
            }

Reply:
@bxf12315  sorry I misunderstood the behavior.. this navigation have to be present after the pi is not active.. forget this comment!